app-tracking-protection/vpn-impl/src/main/java/com/duckduckgo/mobile/android/vpn/service/TrackerBlockingVpnService.kt

```diff
@@ -285,6 +286,8 @@ class TrackerBlockingVpnService : VpnService(), CoroutineScope by MainScope() {
                 }
             }
 
+            vpnNetworkStack.dns().forEach { addDnsServer(it) }
+
```

Comment:
I would add this inside the `getDns()` method. When we set the DNS we do several checks that I think we should still do here.